api/src/main/java/com/velocitypowered/api/proxy/server/ServerPing.java

```diff
@@ -412,6 +412,8 @@ public int hashCode() {
 
   public static final class SamplePlayer {
 
+    public static final SamplePlayer Anonymous = new SamplePlayer("Anonymous Player", new UUID(0L, 0L));
```

Comment:
The last minor nit: constants  are `UPPER_CASE`.

Reply:
Fixed